swap/src/asb/config.rs

```diff
@@ -226,6 +276,9 @@ mod tests {
                 wallet_rpc_url: Url::from_str(DEFAULT_MONERO_WALLET_RPC_TESTNET_URL).unwrap(),
             },
             tor: Default::default(),
+            logging: Logging {
+                format: LoggingFormat::Plain,
+            },
```

Comment:
Does this have to go in the config? I think a command-line switch would be sufficient no?

Reply:
Happy to do so, that could solve my issue I think.